### ISSUE TITLE
Example values in Swagger Docs not properly generated from aliases.

### ISSUE THREAD

**Issue by robinsandstrom** [vitalik/django-ninja#131]
Aliases is not rendered properly in example value for request body:

```
from ninja import Schema
from humps import camelize

def to_camel(string: str) -> str:
    return camelize(string)

class CamelCaseSchema(Schema):
    class Config:
        alias_generator = to_camel
        allow_population_by_field_name = True

```
Expected behaviour in request body example value:

`
{
 "zipCode": "string"
}
`

Actual behaviour:

`
{
 "zip_code": "string"
}
`

Which makes it confusing since the actual field the api is expecting is a camelcased value. I tried fiddling with the OpenAPI-Class but got nowhere unfortunately. Thank you for an otherwise great package, looking forward on implementing it in a project of ours!

I found this link for [FastAPI](https://medium.com/analytics-vidhya/camel-case-models-with-fast-api-and-pydantic-5a8acb6c0eee) where it seems to be working as expected.

**Comment by vitalik** [comment]
Hi @robinsandstrom 


to use alias_generator you need to serialize pydantic model with `by_alias=True`

django ninja allows you to pass this argument in method decorator:


```Python
....
class CamelCaseSchema(Schema):
    some_key: str

    class Config:
        alias_generator = to_camel
        allow_population_by_field_name = True


@api.post("/test", response=CamelCaseSchema, by_alias=True)
def test_custom(request):
    return {"some_key": "some_value"}
```

result

```JSON
{
  "someKey": "some_value"
}
```

**Comment by robinsandstrom** [comment]
Hi! I can submit a more thorough explanation later, but somehow the aliasing is not working when inheriting classes multiple times, i.e.

`class CamelCaseSchema(Schema):
    some_key: str

    class Config:
        alias_generator = to_camel
        allow_population_by_field_name = True

class UserProxy(CamelCaseSchema):
    user_id: str
 
class UserRequest(UserProxy):
   password: str

class UserResponse(UserProxy):
   client_id: str
`

Might be a pydantic issue, but I will look at it more to see what's going on.

**Comment by vitalik** [comment]
@robinsandstrom well it works for me

are you sure did not forgot to pass `by_alias=True`

this is my working case:
`@api.post("/test", response=UserResponse, by_alias=True)`

**Comment by robinsandstrom** [comment]
Somehow it's working on half of the schemas inherited by the base camel-case schema, but not on all of them:

```
class CompanyProxy(Schema):
    id: UUID = None
    name: str
    company_id: str


class CompanyRequest(CompanyProxy):
    pass


class CompanyCreateRequest(CompanyRequest):
    pass


class CompanyUpdateRequest(CompanyRequest, optional_fields="__all__"):
    pass


class CompanyResponse(CompanyProxy):
    id: UUID


class CompanyHierarchy(CompanyProxy):
    children: List['CompanyHierarchy'] = []
```

and apis:

```
@api.get('/', response=List[schemas.CompanyResponse])
def list(request):
    qs = models.Company.objects.all()
    return qs


@api.post("/", response=schemas.CompanyResponse, by_alias=True)
def create(request, payload: schemas.CompanyCreateRequest):
    company = models.Company.objects.create(
        **payload.dict())
    return company
```

Showing: 

<img width="1271" alt="Skärmavbild 2021-05-06 kl  15 51 54" src="https://user-images.githubusercontent.com/16864747/117309734-fdaeb980-ae82-11eb-9995-9f9128548f47.png">

**Comment by javierhuerta** [comment]
Hi, thanks for django ninja it's super cool. The same thing happens to me but the alias conversion is done correctly, I think it only happens that in the api documentation what is related to the ResponseModel does not show the alias.

**Comment by elvisgs** [comment]
I think the reason is [here](https://github.com/vitalik/django-ninja/blob/master/ninja/openapi/schema.py#L186). When creating schema for responses, `by_alias` is explicitly disabled. Although, when creating schema for body [here](https://github.com/vitalik/django-ninja/blob/master/ninja/openapi/schema.py#L155), `by_alias`  is default `True`.

**Comment by stephenrauch** [comment]
@vitalik I just verified that this was fixed in 0.15.0

**Comment by vitalik** [comment]
@robinsandstrom 
yep, seems fixed - feel free to reopen if you see an issue
